### PR TITLE
Allow to give file name parameters

### PR DESCRIPTION
**Pull request by elodieirdor**
Hello,
I would like to personalize the filename of my .env and .env.dist. How do you think of this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,24 @@ You can run Companienv automatically after `composer install` or `composer updat
 }
 ```
 
+The ``.env`` will then be created or updated by the composer script, to match the structure of the dist file .env.dist by asking you the missing parameters.
```
Comment (sroze) [inline README.md]: I guess you could remove this sentence, it's a bit duplicated from the other context I believe.

File: README.md [hunk 2]
```diff
@@ -43,6 +43,24 @@ You can run Companienv automatically after `composer install` or `composer updat
...
+
+By default, the files are assumed to be .env.dist and .env. This can be changed in the configuration:
```
Comment (sroze) [inline README.md]: What about this?

> By default, the file used as a template is `.env.dist` and the written file is `.env`. You can change these defaults within your `composer.json` file:

File: README.md [hunk 3]
```diff
@@ -43,6 +43,24 @@ You can run Companienv automatically after `composer install` or `composer updat
...
+
+
+```json
+{
+    "extra": {
+        "companienv-parameters": [
+          {
```
Comment (sroze) [inline README.md]: Could you fix the indentation here?

File: src/Companienv/Composer/ScriptHandler.php [hunk 4]
```diff
@@ -6,22 +6,31 @@
...
-        );
-        $companion->fillGaps();
+        if (isset($extras['companienv-parameters'])) {
+            $configs = $extras['companienv-parameters'];
+        } else {
+            $configs = [['file' => Application::defaultFile(), 'dist-file' => Application::defaultDistributionFile()]];
+        }
+
+        foreach ($configs as $config) {
+            $directory = getcwd();
```
Comment (sroze) [inline src/Companienv/Composer/ScriptHandler.php]: This can be moved outside of the `foreach`

**Review by sroze — CHANGES_REQUESTED** [review]
I like the idea of configuring these files name, thank you.

Do you actually have the need for changing multiple `.env` files at the same time with the Composer integration?

**Comment by elodieirdor** [comment]
Yeah, we actually need this, we store the parameters for our different language in different files. We use https://github.com/Incenteev/ParameterHandler for our project with symfony from 2.x to 3.x.

File: README.md [hunk 5]
```diff
@@ -43,20 +43,18 @@ You can run Companienv automatically after `composer install` or `composer updat
 }
 ```
 
-The ``.env`` will then be created or updated by the composer script, to match the structure of the dist file .env.dist by asking you the missing parameters.
-
-By default, the files are assumed to be .env.dist and .env. This can be changed in the configuration:
+By default, the file used as a template is `.env.dist` and the written file is `.env`. You can change these defaults within your composer.json file:
```
Comment (sroze) [inline README.md]: Missing \` around the `composer.json` :)

Comment (elodieirdor) [reply README.md]: ^^ my bad

**Review by sroze — APPROVED** [review]
Super, thank you!

**Comment by elodieirdor** [comment]
Great! Thank you !!

**Pull request merged**